packages/twenty-front/src/modules/workflow/workflow-diagram/components/WorkflowRunDiagramCanvasEffect.tsx

```diff
@@ -24,27 +24,29 @@ export const WorkflowRunDiagramCanvasEffect = () => {
 
   const workflowId = useRecoilValue(workflowIdState);
 
-  const { activeTabIdState: workflowRunRightDrawerListActiveTabIdState } =
-    useTabListStates({
-      tabListScopeId: WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID,
-    });
-
   const goBackToFirstWorkflowRunRightDrawerTabIfNeeded = useRecoilCallback(
     ({ snapshot, set }) =>
       () => {
         const activeWorkflowRunRightDrawerTab = getSnapshotValue(
           snapshot,
-          workflowRunRightDrawerListActiveTabIdState,
+          activeTabIdComponentState.atomFamily({
+            instanceId: WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID,
```

Comment:
Why did we keep this `WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID` instead of using `TAB_LIST_COMPONENT_ID` from `ShowPageTabListComponentId`?